projboard/tests/test_client.py

```diff
@@ -97,14 +98,26 @@ def test_fill_article_delete(client):
 
 
 @pytest.mark.django_db
-def test_get_my_articles(client):
+def test_get_my_articles(client, user, user_articles):
+    # test edge case of sending user nickname = ''
     response = client.get("/my_articles/")
+    assert response.status_code == 404
```

Comment:
This should move to different test
But it seems you really understand the point right now
Thanks

Please change this

Reply:
done